### PR TITLE
RHD-594 - Added in JIRA issue collector

### PR DESCRIPTION
**Pull request by Dantheman720**

### CODE REVIEW

File: _layouts/base.html.slim [hunk 1]
```diff
@@ -247,6 +247,7 @@ body(class="#{page.url.gsub('/','').split('.').first || 'home'} #{page.body_clas
     script src="#{site.base_url}/javascripts/experiments.js"
     script src="#{site.base_url}/javascripts/overlay.js"
     script src="#{site.base_url}/javascripts/devnation2015.js"
+    script src="#{site.base_url}/javascripts/jira-collector.js"
```
Comment (LightGuard) [inline _layouts/base.html.slim]: Do we need to load this everywhere?

Comment (paulrobinson) [inline _layouts/base.html.slim]: It appears on every page.

**Comment by catrobson** [comment]
Looks great to me.  James might want to adjust the visual design but I'm good with it as is.

**Comment by Dantheman720** [comment]
Can one of the admins verify this patch?

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
This needs to be rebased.

**Comment by paulrobinson** [comment]
@Dantheman720 can you rebase. Apologies, I missed @catrobson's feedback an would have merged sooner.

**Pull request merged**